Assets/MixedRealityToolkit-SDK/Features/Boundary/Components/Scripts/FloorPlaneVisualizer.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.BoundarySystem;
+using Microsoft.MixedReality.Toolkit.Internal.Managers;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SDK.Boundary
+{
+    /// <summary>
+    /// Visualizes the floor plane.
+    /// </summary>
+    public class FloorPlaneVisualizer : MonoBehaviour
+    {
+        [Tooltip("Should the floor plane be visualized?")]
+        [SerializeField]
+        private bool isFloorPlaneVisualized = true;
+
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the floor plane is / should be visualized.
+        /// </summary>
+        public bool IsFloorPlaneVisualized
+        {
+            get { return isFloorPlaneVisualized; }
+            set
+            {
+                if (value != isFloorPlaneVisualized)
+                {
+                    isFloorPlaneVisualized = value;
+                }
+            }
+        }
+
+        private MixedRealityBoundaryVisualizationProfile visualizationProfile;
```

Comment:
Why do we need to keep a reference to this?  It should only ever be read from the Active Profile.

Reply:
I'd even argue that components should never really be reading from the profile, as it should generally be consumed by managers that are implementing them.

The Managers should be passing this information to any components that need the required information. That way we don't have to query the MR Manager at all but to get the active manager for our system.

Reply:
The boundary manager does provide this data., i cached here since it is the only portion of the boundary system we have been using. BTW, i should probably get the floor Y value from the boundary manager, so this will be changing today